test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresUnreferencedCodeCapability.cs

```diff
@@ -220,6 +221,27 @@ static void TestTypeWhichOverridesMethodVirtualMethodRequiresUnreferencedCodeOnB
 			tmp.VirtualMethodRequiresUnreferencedCode ();
 		}
 
+		class PropertyBaseType
+		{
+			public virtual int VirtualPropertyRequiresUnreferencedCode { [RequiresUnreferencedCode ("Message for --PropertyBaseType.VirtualPropertyRequiresUnreferencedCode--")] get; }
+		}
+
+		class TypeWhichOverridesProperty : PropertyBaseType
+		{
+			public override int VirtualPropertyRequiresUnreferencedCode {
+				[RequiresUnreferencedCode ("Message for --TypeWhichOverridesProperty.VirtualPropertyRequiresUnreferencedCode--")]
+				get { return 1; }
+			}
+		}
+
+		[LogDoesNotContain ("TypeWhichOverridesProperty.VirtualPropertyRequiresUnreferencedCode")]
+		[ExpectedWarning ("IL2026", "--PropertyBaseType.VirtualPropertyRequiresUnreferencedCode--")]
+		static void TestTypeWhichOverridesVirtualPropertyRequiresUnreferencedCode ()
```

Comment:
Is this any different from the `TestTypeWhichOverridesMethodVirtualMethodRequiresUnreferencedCode` test that we already have?

Reply:
This new test does not add a lot of value to the linker since it treats the properties as methods. The analyzer, on the other hand, registers different operation actions depending on if it's a method invocation or a property reference. Therefore the analyzer could handle different properties and methods. For example, the code before this change will warn incorrectly on derived properties, but wouldn't warn on a method invocation.
I think it's important to test it on the side of the analyzer and it doesn't hurt to have it in the linker.